plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergConnectorTest.java

```diff
@@ -203,6 +209,46 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Override
+    protected void checkInformationSchemaTablesForPointedQueryForMaterializedView(String schemaName, String viewName)
+    {
+        // TODO The query should not fail, obviously. It should return the viewName, as information_schema.tables returns it when invoked without filters
+        assertThatThrownBy(() -> super.checkInformationSchemaTablesForPointedQueryForMaterializedView(schemaName, viewName))
+                .hasMessageContaining("Not an Iceberg table");
+    }
+
+    @Override
+    protected void checkShowColumnsForMaterializedView(String viewName)
```

Comment:
where are you overriding those?

Reply:
@losipiuk  in `AbstractTestIcebergConnectorTest`

Reply:
@losipiuk sorry, i think i misunderstood the question.